.github/workflows/ci.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     name: NPM Unit Test
     uses: uc-cdis/.github/.github/workflows/npm_unit_test.yaml@master
     with:
-      node-version: '14'
+      node-version: '16'
```

Comment:
ok, this means the following PR can be cancelled: https://github.com/uc-cdis/data-portal/pull/1222/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR34